src/Backend/Modules/Blog/Installer/Installer.php

```diff
@@ -198,27 +201,24 @@ private function getSearchWidgetId(): int
 
     private function hasPageWithBlogBlock(string $language): bool
     {
-        // @todo: Replace with a PageRepository method when it exists.
-        return (bool) $this->getDatabase()->getVar(
-            'SELECT 1
-             FROM PagesPage AS p
-             INNER JOIN PagesPageBlock AS b ON b.revision_id = p.revision_id
-             WHERE b.extra_id = ? AND p.language = ?
-             LIMIT 1',
-            [$this->blogBlockId, $language]
+        return Model::getContainer()->get(PageBlockRepository::class)->moduleExtraExistsForLocale(
+            $this->blogBlockId,
+            Locale::fromString(
+                $language
+            )
         );
     }
 
     /**
      * Insert a category for a language
      *
-     * @todo: Replace this with a BlogCategoryRepository method when it exists.
-     *
```

Comment:
Why not remove the entire docblock except for this comment while you're here? It doesn't provide any useful additional information

Reply:
fixed in c9b52a0b0